### PR TITLE
Handle erroneous arguments

### PR DESCRIPTION
**Pull request by S2dentik**
```
Handle the case when Taylor is invoked with invalid arguments:
* Remove force try inside `processArguments()` that was leading to
imminent crash and unclear error message.
* If an invalid parameter has been sent print error message and
prompt the user to play Pacman.
```

### CODE REVIEW

**Comment by S2dentik** [comment]
Depends on #27, that one will need to be accepted first then this one rebased.

File: TaylorFrameworkTests/CapriceTests/Mocks/MockMessageProcessor.swift [hunk 1]
```diff
@@ -42,7 +42,7 @@ class MockMessageProcessor : MessageProcessor {
         
         if arguments.count.isOdd {
             let optionsProcessor = MockOptionsProcessor()
-            return optionsProcessor.processOptions(arguments)
+            return try! optionsProcessor.processOptions(arguments)
```
Comment (thelvis4) [inline TaylorFrameworkTests/CapriceTests/Mocks/MockMessageProcessor.swift]: Please add a message why force try is necessary here.

**Comment by thelvis4** [comment]
`* If an invalid parameter has been sent print error message and prompt the user to play Pacman.` this is still not working.

**Comment by S2dentik** [comment]
Fixed and rebased. Ready to be looked at.

**Comment by thelvis4** [comment]
If you pass `taylor -c sdf` and give `YES`, we get `[1]    16150 illegal hardware instruction  taylor -c sdf` which doesn't seem to be the correct behaviour. :confused:

**Comment by thelvis4** [comment]
Still reproducing

**Comment by S2dentik** [comment]
Fixed in #30

**Comment by thelvis4** [comment]
Taylor thanks you! 👧

**Pull request merged**